src/main/java/org/codehaus/mojo/versions/RevertMojo.java

```diff
@@ -48,24 +56,62 @@
     @Parameter( defaultValue = "${project}", required = true, readonly = true )
     private MavenProject project;
 
-    public void execute()
-        throws MojoExecutionException, MojoFailureException
+    /**
+     * Whether to start processing at the local aggregation root (which might be a parent module
+     * of that module where Maven is executed in, and the version change may affect parent and sibling modules).
+     * Setting to false makes sure only the module (and it's submodules) where Maven is executed for is affected.
+     *
+     * @since 2.13.0
+     */
+    @Parameter( property = "processFromLocalAggregationRoot", defaultValue = "true" )
```

Comment:
Why not the same name as for set goal - `processAllModules`?

Reply:
Because it's an analogue of `processFromLocalAggregationRoot` from `SetMojo`, that's why it has the same name as that property :)